tsconfig.json

```diff
@@ -1,13 +1,17 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2018",
```

Comment:
So if I don't do this, it simply does not work, didn't investigate further, is it a problem to change it you think?